packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -52,6 +52,13 @@ import {
   requestsToKilosecond,
 } from './utils';
 
+let CID: any;
+try {
+  CID = require('multiformats/cid');
+} catch (e) {
+  console.log('CID not found');
+}
+
```

Comment:
For context this is the original implementation which does correctly transpile. We are essentially doing a revert with pr to an earlier implementation which could transpile to all of our required targets.